testsuite/tests/conftest.py

```diff
@@ -256,12 +255,6 @@ def kuadrant(request, testconfig, openshift):
     if request.config.getoption("--standalone"):
         return None
 
-    # Try if Kuadrant is deployed
-    kuadrant_openshift = testconfig["service_protection"]["system_project"]
-    kuadrants = kuadrant_openshift.do_action("get", "kuadrant", "-o", "json", parse_output=True)
-    if len(kuadrants.model["items"]) == 0:
-        pytest.fail("Running Kuadrant tests, but Kuadrant resource was not found")
-
     # TODO: Return actual Kuadrant object
     return True
```

Comment:
Looks like you don't need `testconfig` and `openshift` fixtures here anymore (badly placed unused-parameter 😬)

Reply:
Removed the fixtures, and kept the unused-argument. There were other issue related to that